packages/sdk-react-native/ios/MetaMaskReactNativeSdk.swift

```diff
@@ -41,18 +41,11 @@ class MetaMaskReactNativeSdk: NSObject, RCTBridgeModule {
       sdkOptions = SDKOptions(infuraAPIKey: infuraAPIKey)
     }
 
-    // Logging the API version and adding it to the SDK options if necessary
-    // if let apiVersion = options.apiVersion {
-    //   Logging.log("MetaMaskReactNativeSdk:: API version: \(apiVersion)")
-    //   sdkOptions?.apiVersion = apiVersion // Assuming SDKOptions has an apiVersion field
-    // }
-
     metaMaskSDK = MetaMaskSDK.shared(
       AppMetadata(
         name: options.dappName,
         url: options.dappUrl,
-        iconUrl: options.dappIconUrl,
-        apiVersion: options.apiVersion),
+        iconUrl: options.dappIconUrl),
```

Comment:
What's the reason to this change, Maybe by mistake?
Pls Undo all this changes to the `sdk-react-native`